src/audio/volume/volume.c

```diff
@@ -212,7 +222,7 @@ static int32_t volume_windows_fade_ramp(struct comp_dev *dev, int32_t ramp_time,
 	int32_t pow_value; /* Q2.30 */
 	int32_t volume_delta = cd->tvolume[channel] - cd->rvolume[channel]; /* Q16.16 */
 
-	time_ratio = (((int64_t)ramp_time) << 30) / (cd->ipc_config.initial_ramp << 3);
+	time_ratio = (((int64_t)ramp_time) << 30) / (cd->initial_ramp << 3);
```

Comment:
Does this break windows fade ramp type for IPC3? It should be available for IPC3 too, it's only a token set in topology.

Reply:
Sorry, looks OK -> resolved